developer/src/kmc/build.sh

```diff
@@ -51,40 +51,48 @@ fi
 if builder_start_action clean; then
   rm -rf ./build/ ./tsconfig.tsbuildinfo
   builder_finish_action success clean
-else
+fi
+
+function copy_schemas() {
```

Comment:
1. This makes the diff easy to see.  Just checking if you wanted to move the function higher to keep all the builder actions together.
2. Do you want to use internal name  `_copy_schemas()`?

Reply:
No, I think it's fine there for now; no need to use `_` here because this script isn't imported into other scripts like builder.inc.sh is.